commands/comment.go

```diff
@@ -53,14 +58,23 @@ func commentHashExists(hashToFind string, threads []review.CommentThread) bool {
 }
 
 // checkCommentLocation verifies that the given location exists at the given commit.
-func checkCommentLocation(repo repository.Repo, commit, file string, line uint) error {
+func checkCommentLocation(repo repository.Repo, commit, file string, location comment.Range) error {
 	contents, err := repo.Show(commit, file)
 	if err != nil {
 		return err
 	}
 	lines := strings.Split(contents, "\n")
-	if line > uint(len(lines)) {
-		return fmt.Errorf("Line number %d does not exist in file %q", line, file)
+	if location.StartLine > uint32(len(lines)) {
+		return fmt.Errorf("Line number %d does not exist in file %q", location.StartLine, file)
+	}
+	if location.StartColumn != 0 && location.StartColumn > uint32(len(lines[location.StartLine])) {
```

Comment:
Since we are numbering lines starting at 1, we need to subtract 1 from location.StartLine before indexing into the lines slice.

We also need to do that below when checking the location.EndColumn field.